hw/rdma/rdma_backend.c

```diff
@@ -248,20 +248,21 @@ static int rdmacm_mux_send(RdmaBackendDev *backend_dev, RdmaCmMuxMsg *msg)
     rc = qemu_chr_fe_write(backend_dev->rdmacm_mux.chr_be,
                            (const uint8_t *)msg, sizeof(*msg));
     if (rc != sizeof(*msg)) {
-        enable_rdmacm_mux_async(backend_dev);
         rdma_error_report("Failed to send request to rdmacm_mux (rc=%d)", rc);
-        return -EIO;
+        rc = -EIO;
+        goto out;
     }
 
     rc = rdmacm_mux_check_op_status(backend_dev->rdmacm_mux.chr_be);
     if (rc) {
         rdma_error_report("Failed to execute rdmacm_mux request %d (rc=%d)",
                           msg->hdr.op_code, rc);
+        goto out;
```

Comment:
This changes the behaviour of the function.
I really do not remember why in case that status validation fails i still returning 0 to caller, have to check it but anyway, your patch is not a cosmetic one - it is actually changing the behaviour of the function.

Reply:
So, you probably need to document that to avoid confusion.